mlir/include/mlir/Dialect/Transform/Transforms/Passes.td

```diff
@@ -51,8 +51,9 @@ def PreloadLibraryPass : Pass<"transform-preload-library"> {
 
     Warning: Only a single such pass should exist for a given MLIR context.
     This is a temporary solution until a resource-based solution is available.
-    TODO: investigate using a resource blob if some ownership mode allows it.
   }];
+  // TODO: investigate using a resource blob if some ownership mode allows it.
+  let dependentDialects = ["transform::TransformDialect"];
```

Comment:
Nit: use fully-qualified names in .td, e.g., `::mlir::transform::TransformDialect`.